src/lib/schematics/ng-generate/nav/schema.json

```diff
@@ -55,13 +55,13 @@
       "description": "The prefix to apply to generated selectors.",
       "alias": "p"
     },
-    "styleext": {
+    "style": {
       "description": "The file extension to be used for style files.",
       "type": "string"
     },
-    "spec": {
+    "skipTests": {
       "type": "boolean",
-      "description": "Specifies if a spec file is generated."
+      "description": "Whether a test file should be generated."
```

Comment:
Nit: ``When true, does not create \"spec.ts\" test files for the app`

Reply:
Improved that description. thanks!